test_scripts/PROJECTION/004_OnTouchEvent_while_streamings.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- In case:
+-- 1) Application is registered with PROJECTION appHMIType
+-- 2) and starts video and audio streaming
+-- 3) HMI sends OnTouchEvent
+-- SDL must:
+-- 1) successful transfer notifications to mobile application
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/PROJECTION/common')
+local runner = require('user_modules/script_runner')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local appHMIType = "PROJECTION"
+local OnTochEventType = {
```

Comment:
@GetmanetsIrina What is `OnToch`? May be `OnTouch`?

Reply:
@AByzhynar , you are right, locally changed.